examples/acados_matlab_octave/test/test_all_examples.m

```diff
@@ -35,6 +35,7 @@
 % list the examples you would like to test
 
 targets = {
+    '../control_rates/main.m';
```

Comment:
It seems like this is failing.
But the fix might be contained in https://github.com/acados/acados/pull/1320/files

Reply:
Yes, I think I will wait until it's merged :)